### PR TITLE
Querying-best-practices edits 

### PR DESCRIPTION
**Pull request by idalithb**
-Edits

### CODE REVIEW

File: website/pages/en/querying/querying-best-practices.mdx [hunk 1]
```diff
@@ -2,11 +2,9 @@
 title: Querying Best Practices
 ---
 
-The Graph provides a decentralized way to query data from blockchains.
+The Graph provides a decentralized way to query data from blockchains via GraphQL API, making it easier to query data with the GraphQL language.
```
Comment (benface) [inline website/pages/en/querying/querying-best-practices.mdx:5]:
```suggestion
The Graph provides a decentralized way to query data from blockchains via GraphQL APIs, making it easier to query data with the GraphQL language.
```
Also, I find it interesting that we don't mention the word "subgraph" here... those GraphQL APIs are subgraphs :P

File: website/pages/en/querying/querying-best-practices.mdx [hunk 2]
```diff
@@ -2,11 +2,9 @@
...
 
-The Graph network's data is exposed through a GraphQL API, making it easier to query data with the GraphQL language.
-
-This page will guide you through the essential GraphQL language rules and GraphQL queries best practices.
+Learn the essential GraphQL language rules and GraphQL queries best practices.
```
Comment (benface) [inline website/pages/en/querying/querying-best-practices.mdx:7]:
```suggestion
Learn the essential GraphQL language rules and best practices around querying with GraphQL.
```

File: website/pages/en/querying/querying-best-practices.mdx (hunk 1 above)
Comment (idalithb) [reply website/pages/en/querying/querying-best-practices.mdx:5]: Not sure if this was intentional (and if it was, don't know for what reason)

File: website/pages/en/querying/querying-best-practices.mdx [hunk 3]
```diff
@@ -191,17 +187,18 @@ const result = await execute(query, {
...
 
-When querying GraphQL APIs, always think of querying only the fields that will be actually used.
-
-A common cause of over-fetching is collections of entities. By default, queries will fetch 100 entities in a collection, which is usually much more than what will actually be used, e.g., for display to the user. Queries should therefore almost always set first explicitly, and make sure they only fetch as many entities as they actually need. This applies not just to top-level collections in a query, but even more so to nested collections of entities.
+- When querying GraphQL APIs, always think of querying only the fields that will be actually used.
+- A common cause of over-fetching is collections of entities. By default, queries will fetch 100 entities in a collection, which is usually much more than what will actually be used, e.g., for display to the user.
+- Queries should always be set first explicitly.
+  - Make sure queries only fetch as many entities as they actually need. This applies not just to top-level collections in a query, but even more so to nested collections of entities.
```
Comment (benface) [inline website/pages/en/querying/querying-best-practices.mdx:201]: A bit repetitive. I propose:

```suggestion
- Make sure queries only fetch as many entities as you actually need. By default, queries will fetch 100 entities in a collection, which is usually much more than what will actually be used, e.g., for display to the user. This applies not just to top-level collections in a query, but even more so to nested collections of entities.
```

File: website/pages/en/querying/querying-best-practices.mdx (hunk 2 above)
Comment (idalithb) [reply website/pages/en/querying/querying-best-practices.mdx:7]:
```suggestion
Learn the essential GraphQL language rules and GraphQL querying best practices.
```

File: website/pages/en/querying/querying-best-practices.mdx [hunk 4]
```diff
@@ -337,8 +334,8 @@ query {
 
 Such repeated fields (`id`, `active`, `status`) bring many issues:
 
-- harder to read for more extensive queries
-- when using tools that generate TypeScript types based on queries (_more on that in the last section_), `newDelegate` and `oldDelegate` will result in two distinct inline interfaces.
+- Harder to read for more extensive queries
```
Comment (benface) [inline website/pages/en/querying/querying-best-practices.mdx:337]:
```suggestion
- More extensive queries become harder to read.
```

File: website/pages/en/querying/querying-best-practices.mdx [hunk 5]
```diff
@@ -411,16 +408,16 @@ fragment VoteItem on Vote {
 
 It is not possible to spread a fragment of type `Vote` here.
 
-**Define Fragment as an atomic business unit of data**
+#### Define Fragment as an atomic business unit of data
 
 GraphQL Fragment must be defined based on their usage.
```
Comment (benface) [inline website/pages/en/querying/querying-best-practices.mdx:413]:
```suggestion
GraphQL `Fragment`s must be defined based on their usage.
```

File: website/pages/en/querying/querying-best-practices.mdx [hunk 6]
```diff
@@ -411,16 +408,16 @@ fragment VoteItem on Vote {
...
 
 For most use-case, defining one fragment per type (in the case of repeated fields usage or type generation) is sufficient.
 
 Here is a rule of thumb for using Fragment:
```
Comment (benface) [inline website/pages/en/querying/querying-best-practices.mdx:417]:
```suggestion
Here is a rule of thumb for using fragments:
```

File: website/pages/en/querying/querying-best-practices.mdx [hunk 7]
```diff
@@ -411,16 +408,16 @@ fragment VoteItem on Vote {
...
 
-- when fields of the same type are repeated in a query, group them in a Fragment
-- when similar but not the same fields are repeated, create multiple fragments, ex:
+- When fields of the same type are repeated in a query, group them in a Fragment
+- When similar but not the same fields are repeated, create multiple fragments, ex:
```
Comment (benface) [inline website/pages/en/querying/querying-best-practices.mdx:420]:
```suggestion
- When fields of the same type are repeated in a query, group them in a `Fragment`.
- When similar but different fields are repeated, create multiple fragments, for instance:
```

**Review by benface — APPROVED** [review]
🙌

**Pull request merged**